tasks.py

```diff
@@ -26,41 +30,73 @@
     "deploy_path": SETTINGS["OUTPUT_PATH"],
     # Github Pages configuration
     "github_pages_branch": "gh-pages",
-    "commit_message": "'Publish site on {}'".format(datetime.date.today().isoformat()),
+    "commit_message": "'Publish site on {}'".format(date.today().isoformat()),
     # Host and port for `serve`
     "host": "localhost",
     "port": 8000,
 }
 
+# templates for creating a new post
+
+POST_TEMPLATE = dedent(
+    """
+    Title: {title}
+    Date: {date}
+    Modified: {modified}
+    Category: {category}
+    Tags: {tags}
+    Slug: {slug}
+    Authors: {authors}
+    Summary: {summary}
+
+    [TOC]
+
+    ---
+    <!--your content here-->
+"""
+)
```

Comment:
`.strip()` should be added to the end to prevent an additional empty line above `Title`. It's my bad for missing this 😂

Reply:
Instead of using `strip()`, I removed the line above. Please take another look. Thanks!